Documentation/PageTsconfig/TceForm.rst

```diff
@@ -63,1113 +65,1206 @@ dots ('.'), these must be escaped with backslash.
 
-renderFunc
-~~~~~~~~~~
+    TCEFORM.suggest.default {
+        # sets the pidList for a suggest fields in all tables
+        pidList = 1,2,3,45
+    }
 
-:aspect:`Datatype`
-   string
 
-:aspect:`Description`
-   User function to manipulate the displayed records in the result.
+..  index::
+    Suggest wizard; receiver php class
+..  _tceform-suggest.receiverClass:
 
+suggest.receiverClass
+~~~~~~~~~~~~~~~~~~~~~
 
-.. index::
-   Suggest wizard; Where statement
+..  confval:: suggest.receiverClass
+    :name: tceform-suggest-receiverClass
+    :type: Fully Qualified PHP class name
+    :Default: :php:`\TYPO3\CMS\Backend\Form\Element\SuggestDefaultReceiver`
 
-searchCondition
-~~~~~~~~~~~~~~~
+    PHP class alternative receiver class - the file that holds the class should be derived
+    from :code:`\TYPO3\CMS\Backend\Form\Element\SuggestDefaultReceiver`.
 
-:aspect:`Datatype`
-   string
+..  index::
+    Suggest wizard; rendering user function
+..  _tceform-suggest-renderFunc:
 
-:aspect:`Description`
-   Additional WHERE clause (no AND needed to prepend).
+suggest.renderFunc
+~~~~~~~~~~~~~~~~~~
 
-:aspect:`Example`
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/page.tsconfig
+    :name: tceform-suggest.renderFunc
+..  confval::
+    :name: tceform-suggest-renderFunc
+    :type: string
 
-      TCEFORM.pages.storage_pid.suggest.default {
-         # Configure the suggest wizard for the field "storage_pid" in table "pages"
-         # to search only for pages with doktype=1
-         searchCondition = doktype=1
-      }
+    User function to manipulate the displayed records in the result.
 
-.. index::
-   Suggest wizard; Search whole phrase
+..  index::
+    Suggest wizard; Where statement
+..  _tceform-suggest-searchCondition:
 
-searchWholePhrase
-~~~~~~~~~~~~~~~~~
+suggest.searchCondition
+~~~~~~~~~~~~~~~~~~~~~~~
 
-:aspect:`Datatype`
-   boolean
+..  confval::
+    :name: tceform-suggest.searchCondition
+    :type: string
 
-:aspect:`Description`
-   Whether to do a `LIKE=%mystring%` (searchWholePhrase = 1) or a `LIKE=mystring%` (to do a real find as you type).
+    Additional WHERE clause (no AND needed to prepend).
+
+..  _tceform-suggest-searchCondition-example:
+
+Example:Only search on pages with doktype=1
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    ..  code-block:: typoscript
+        :caption: EXT:site_package/Configuration/page.tsconfig
+
+        TCEFORM.pages.storage_pid.suggest.default {
+            # Configure the suggest wizard for the field "storage_pid" in table "pages"
+            # to search only for pages with doktype=1
+            searchCondition = doktype=1
+        }
+
+..  index::
+    Suggest wizard; Search whole phrase
+..  _tceform-suggest-searchWholePhrase:
+
+suggest.searchWholePhrase
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  confval:: suggest.searchWholePhrase
+    :name: tceform-suggest-searchWholePhrase
+    :type: boolean
+    :Default: false
```

Comment:
```suggestion
    :Default: 0
```

We deal here with TSconfig, not with PHP. I am not sure, if "false" is really evaluated to false (or to true, as this is a non-empty string).

Reply:
We have this thing all over the guide, once we write false, once we write 0 We should make a central PR for this. If we decide for search and replace, replacing :Default: false with :Default: 0 is easier then vize versa however.

Reply:
You change from "0" to "false" in your PR. So, focus on this PR without changing meaning.